src/airport.js

```diff
@@ -1,6 +1,44 @@
+const Plane = require(`./Plane.js`);
+
 class Airport {
-  // here's a starting point for you
 
-}
+    constructor(maximumCapacity = 10) {
+        this.airportPlanes = [];
+        this.maximumCapacity = maximumCapacity;
+    };
+
+    landPlane = plane => {
+        if (this.airportPlanes.length >= this.maximumCapacity) {
+            return this.airportPlanes.length;
+        }
+        else if (plane instanceof Plane && !this.planeAtAirport(plane)) {
+            this.airportPlanes.push(plane);
+        };
+
+    };
+
+    takeoffPlane = plane => {
```

Comment:
This method I had the most trouble understanding properly. I spent longer than I'd like to admit trying to understand exactly how values were being passed around and/or assigned. I don't think I would've naturally done it this way if similar methods in the basket challenge hadn't been shown to us in this form because it ended up impeding my understanding for a while.